benefits/core/migrations/0070_remove_transitagency_staff_group.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 5.2.7 on 2025-12-15 22:09
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("auth", "0012_alter_user_first_name_max_length"),
+        ("core", "0069_refactor_eligibilityapiverificationrequest"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="transitagency",
+            name="staff_group",
+        ),
+        migrations.AlterField(
+            model_name="transitagency",
+            name="customer_service_group",
+            field=models.OneToOneField(
+                blank=True,
+                default=None,
+                help_text="The group of users who are allowed to do in-person eligibility verification and enrollment.",
+                null=True,
+                on_delete=django.db.models.deletion.PROTECT,
+                related_name="transit_agency",
```

Comment:
Is this line the change? I wasn't expecting an `AlterField` so just wondering what changed.